client/packages/android/app/src/main/assets/capacitor.config.json

```diff
@@ -1,9 +1,16 @@
 {
 	"appId": "org.openmsupply.client",
 	"appName": "openmsupply-client",
-	"webDir": "packages/host/dist",
+	"webDir": "notexistent",
```

Comment:
This file is generated by `npx cap copy`, see capacitor.config.ts below for comments